Cargo.toml

```diff
@@ -5,17 +5,20 @@ authors = ["authors"]
 edition = "2021"
 
 [dependencies]
-bitflags = "1.3"
+bitflags = "2.3"
```

Comment:
did you happen to bump all libraries? `cargo upgrade --all`

Reply:
I have bumped the versions once again with `cargo upgrade`.